extensions/stackdriver/BUILD

```diff
@@ -35,7 +35,7 @@ envoy_cc_library(
     deps = [
         "//extensions/stackdriver/common:context",
         "//extensions/stackdriver/config:stackdriver_plugin_config_cc_proto",
-        "//extensions/stackdriver/metric:registry",
+        "//extensions/stackdriver/metric",
```

Comment:
:metric can be omit? TIL

Reply:
Yeah, if the package name is a dup with the folder name, it gets omitted.